code/c/types/lights.cpp

```diff
@@ -56,23 +57,23 @@ bitmap light_bitmap(light &l)
     switch (l.size)
     {
         case SMALL_LIGHT:
-            strncat(name, "small light", 11);
+            strncat(name, "small light", 14);
```

Comment:
I think this could potentially copy 3 bytes over the end of the `small light` string since `strlen("small light")` = 11.

Since it's a constant string, pretty sure you can use `sizeof()`.

```c++
strncat(name, "small light", sizeof("small light"));
```